netbox/ipam/graphql/types.py

```diff
@@ -250,6 +250,10 @@ class VLANType(NetBoxObjectType):
     interfaces_as_tagged: List[Annotated["InterfaceType", strawberry.lazy('dcim.graphql.types')]]
     vminterfaces_as_tagged: List[Annotated["VMInterfaceType", strawberry.lazy('virtualization.graphql.types')]]
 
+    @strawberry_django.field
```

Comment:
Can this just be done as above in one line, no need for a function?

Reply:
I think it needs to be a function because the type is being used recursively.